frameworks/keyed/angular-cf-new-nozone/src/app/app.component.ts

```
@@ -0,0 +1,87 @@
+    imports: [NgFor],
+    templateUrl: './app.component.html'
+})
+export class AppComponent {
+    data: Array<Data> = [];
+    selected?: number = undefined;
+    id: number = 1;
+    backup?: Array<Data> = undefined;
+    version = VERSION.full;
+
+    buildData(count: number = 1000): Array<Data> {
+        var data: Array<Data> = new Array(count);
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)